packages/go_router_builder/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.11
```

Comment:
This should be in a separate pr. We should avoid touching different packages in same pr, it will make roll back easier if needed to.

Reply:
Faire enough, I've created this PR https://github.com/flutter/packages/pull/2535 on `go_router_builder` and discarded the changes on `go_router_builder` in this PR in  https://github.com/flutter/packages/pull/2444/commits/4aa91506f5f1196e0728a28ef6be91cf47b8bd3c and https://github.com/flutter/packages/pull/2444/commits/66b94dfdc11d7eddffaba6a596db6358f70a415c